e2e-tests/playwright/support/server/default_config.ts

```diff
@@ -60,6 +65,7 @@ const onPremServerConfig = (): Partial<TestAdminConfig> => {
         ServiceSettings: {
             SiteURL: testConfig.baseURL,
             EnableOnboardingFlow: false,
+            GiphySdkKey: 's0glxvzVg9azvPipKxcPLpXV0q1x1fVP',
```

Comment:
I see this value has just been moved, but is it ok to commit? Shouldn't this be a secret?

Reply:
This is a test/dev key only which is also hardcoded on server. I believe it's fine to be exposed here but let me know.
https://github.com/mattermost/mattermost/blob/f5647e0ece3234c20be17f9af97a644772697390/server/public/model/config.go#L112

Reply:
Ah I didn't notice it was hardcoded also on server side. A [quick search](https://github.com/Giphy/giphy-js/issues/120#issuecomment-680988917) shows that it's ok for it to be public, all good on my side then :+1: